app/views/state_file/questions/nj_college_dependents_exemption/edit.html.erb

```diff
@@ -0,0 +1,55 @@
+<%
+    title = t(".title")
+    content_for :page_title, title
+%>
+
+<% content_for :card do %>
+  <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card form-card--long" } do |f| %>
+    <h1 class="h2"><%= title %></h1>
+    <p><%= t(".label_1") %></p>
+    <p><%= t(".label_2") %></p>
+
+    <div class="reveal">
+      <button class="reveal__button"><%= t('.college_helper_heading') %></button>
+      <div class="reveal__content">
+        <p><%= t('.college_helper_description') %></p>
+      </div>
+    </div>
+
+    <div class="reveal">
+      <button class="reveal__button"><%= t('.full_time_college_helper_heading') %></button>
+      <div class="reveal__content">
+        <p><%= t('.full_time_college_helper_description') %></p>
+      </div>
+    </div>
+
+    <div class="reveal">
+      <button class="reveal__button"><%= t('.tuition_books_helper_heading') %></button>
+      <div class="reveal__content">
+        <p><%= t('.tuition_books_helper_description_html') %></p>
+      </div>
+    </div>
+
+    <%= f.fields_for :dependents do |ff| %>
+      <% dependent = ff.object %>
+      <%  if dependent.under_22? %>
+        <div class="white-group">
```

Comment:
[pebble] we're visually grouping each dependent's fields together for sighted users, and should use a fieldset and legend to group them for assistive tech users

Reply:
From slack: we will add a `fieldset` and replace the `h2` with a `legend` element instead